add lookup fields section to docs (#168)
rootstrap/django-drip-campaigns: docs/source/content/create.drip.rst

Here you can relate the Drip to the corresponding ``Campaign``. Grouping several drips under a campaign.

Lookup fields
-------------

> The fields you can make a query are User model fields and fields of models related to the user model. 

Reviewer: ```suggestion
You can make queries using the User's model fields and the fields of the models related to the user.
```

Reviewer: Added this suggestion because "The fields you can make a query" is missing something

Author: Sounds and looks better, thanks!
